examples/plot_rocs.py

```diff
@@ -0,0 +1,137 @@
+"""Produce roc curves from tagger output and labels."""
+import numpy as np
+
+from puma.utils.logging import logger
+from puma.utils import get_dummy_2_taggers
+from puma.metrics import calc_rej
+from puma import roc, roc_plot
+
+# The line below generates dummy data which is similar to a NN output
+df = get_dummy_2_taggers()
+
+logger.info("caclulate tagger discriminants")
+
+
+# define a small function to calculate discriminant
+def disc_fct(a: np.ndarray) -> np.ndarray:
```

Comment:
also here to be consistent, we could add `fc` as argument

Reply:
I added it with `0.018` as default value